apps/fluent-tester/src/TestComponents/Persona/styles.ts

```diff
@@ -1,6 +1,6 @@
 export * from '../PersonaCoin/styles';
 
 export const rajeshImageUrl =
-  'https://3er1viui9wo30pkxh1v2nh4w-wpengine.netdna-ssl.com/wp-content/uploads/prod/2016/09/Rajesh-BW-Square.jpg';
+  'https://media-exp1.licdn.com/dms/image/C4E03AQFH5EV54YFN9g/profile-displayphoto-shrink_800_800/0/1517670612089?e=1673481600&v=beta&t=XaxIkit14fANqZy2jQPlPA03FXhl4iQbF24lq3V2XzM';
```

Comment:
is it possible for us to find safer links, or at least links we explicitly control instead of hard linking to CDNs or sources more liable to changing underneath us?  Perhaps Wikipedia?

Reply:
I found a rajesh link from the seattle times and a mj link from wikipedia. Should those suffice?